src/tests/test_utils.py

```diff
@@ -115,12 +114,6 @@ def test_get_genomes_from_assembly_accession_iterator(self, genome_db_conn):
 					"siteUri": "https://beta.ensembl.org"
 				},
 				"taxon": {
-					"alternativeNames": [
-						"Escherichia coli MG1655",
-						"Escherichia coli str. K12 substr. MG1655",
-						"Escherichia coli str. MG1655",
-						"Escherichia coli strain MG1655",
-					],
```

Comment:
I suspect there might be a hidden issue there. Those values are expected from the taxon test db. Could you check you didn't loose the content from https://github.com/Ensembl/ensembl-metadata-service/pull/43